posts/2021-06-11-kubernetes-operator-with-java-and-quarkus.adoc

```diff
@@ -0,0 +1,712 @@
+= Kubernetes Operator with Java and Quarkus
+
+== The goal of this blog post
+A Kubernetes operator is an application that runs
+Even though Kubernetes provides extensive support for multiple container management operations, it doesn't cover all potential cases related to application management. Some applications need the manual intervention of a human operator. A Kubernetes operator is an application that runs in the cluster and manages resources.
+It allows transferring of operational knowledge of the human operator into software components. Operators can be used for multiple purposes such as restoring data, creating cloud resources, resilience testing, and many others.
+
+The purpose of this blog post is to demonstrate how to quickly write and deploy a simple Kubernetes operator using Java, and provide an example of tasks that can be performed using an operator.
+
+==== Why Java?
+
+* Being able to develop operators in Java can bring a lot of benefits for companies and individuals that are focused mainly on the Java stack.
```

Comment:
Sort of a side note, but depending on how familiar people are with operators this might not be a strong argument. Could I use a simple scripting language instead? Is it solely because a person knows Java that they'd want to do it in Java, or could there be other benifits?

Reply:
The main argument is this could be quite useful for people that are familiar with Java, and don’t want to spend time learning Go or something else before they can start developing operators. I am not trying to prove that Java is a better fit than any other commonly used languages.